docs/howto/openstack/nova/move-instance-between-places.md

```diff
@@ -0,0 +1,107 @@
+- Access to source and target location (RC-file)
+- Enough space on the local machine that you download your server to.
+- Target location configured with Network, Security Groups etc
+
+>To speed up this work, we recommend to use a temporary linux instance located at the same region as the source. This make the transfer faster not needing to downloading to and uploading from your local machine.
+
+## Recommendations
+
+First step will be to create a snapshot of the instance, during this step it is recommended to shutdown the instance to ensure data consistency. Once the snapshot is done the instance can be rebooted again.
+
+Use the UUID of the instance instead of using the instance name, this will make sure you are using the correct instance.
+
+To get the correct UUID of your instance you can use the following command:
+
+```bash
+openstack server list
+```
+
+## Creating a snapshot
+
+To start the snapshot process you use the following command:
+
+```bash
+openstack server image create --wait --name <new-image-name> <uuid-of-instance>
+```
+
+After a while you will get some output showing information of the new image, such as the current state and the image UUID.
+
+Before going on the state needs to be active and you can see the current state with the following command:
+
+```bash
+openstack image show <uuid>
+```
+
+## Download your snapshot
+
+Before you can download the image you need to know what format the image has, to find this out you can run the following command:
+
+```bash
+openstack image show <image_id> | grep disk_format
+```
+
+To download the image to the local machine you use the following command:
+
+```bash
+openstack image save --file <local image name>.<disk_format> <your_image_id>
+```
+
+When the file have been downloaded, you are done with the steps on the source side, and the following steps will be done on the target side.
+
+## Target Location
+
+Load the rc-file for the target side. Instructions how to generate this can be found in the link to OpenStack client installation in the beginning of this guide.
+
+```bash
+source <target_openrc.sh>
+```
+
+## Upload your snapshot
+
+Start upload the file to the new location and use disk_format from above and also the name you want to give the image.
+
+```bash
+openstack image create --disk-format <disk_format> --file snapshot.<disk_format> <new_image_name>
+```
+
+Upload will take a while, and after getting your prompt back, you can check the status by using the following command.
+
+```bash
+openstack image list | grep <new_image_name>
+```
+
+When status is active, the upload is done.
+
+> After you have moved your instance and don't want to keep the old one, don't forget to remove it to save some cost.
+
+## Create a new server
+
+Next step you need to do, is to create a new server from the uploaded image. This can be done either with CLI commands or via the [Control panel](https://cleura.cloud).
```

Comment:
It would be "_the_ next step you need to do", but you can condense this to simply "Next, create a new server from the uploaded image.".

Also, don't hard-code the GUI name and URL. Instead, use macros:

```markdown
This can be done either with CLI commands, or via the [{{extra.gui}}](https://{{extra.gui_domain}}).
```